### PR TITLE
Branch update error messages

### PR DESCRIPTION
**Pull request by KamJiaYue**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/TrendCommand.java [hunk 1]
```diff
@@ -31,7 +31,9 @@ public class TrendCommand extends Command {
      * @param args The year such that the data during this year is to be analyzed.
      */
     public TrendCommand(String args) {
-        this.year = Integer.parseInt(args.trim());
+        String trimmed = args.trim();
```
Comment (laurenlim2112) [inline src/main/java/seedu/address/logic/commands/TrendCommand.java:34]: Good improvement on code quality here! The change is quite minor but makes the code a lot more readable.

File: src/main/java/seedu/address/model/person/Phone.java [hunk 2]
```diff
@@ -11,8 +11,8 @@ public class Phone {
 
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+            "Phone numbers should only contain numbers, and it should be at least 8 digits long";
+    public static final String VALIDATION_REGEX = "\\d{8,}";
```
Comment (laurenlim2112) [inline src/main/java/seedu/address/model/person/Phone.java:15]: Are we allowed to modify this feature's behaviour at this stage? Accepting phone numbers that are <8 digits long might be considered a feature flaw. Feature flaws can be fixed by stating plans to fix them in the Planned Enhancements section of our DG.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -97,12 +97,12 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS g/GENDER l/SEC_LEVEL m/MRT
 
 <box type="tip" seamless>
 
-**Tip:** A student must have at least 1 tag (subject)
+**Tip:** If enrol dates are not specified, it will stick to the default (Today's date)
```
Comment (laurenlim2112) [inline docs/UserGuide.md:100]: I think we might not need to include this tip as it is already stated to the reader above: "When adding subjects, if no enrol date is provided, the date on which the command was executed will be applied to all subjects listed by default."

**Review by laurenlim2112 — COMMENTED** [review]
Good improvements in code quality and test coverage! Might want to take note of some possible feature freeze violations though. Overall, nice work :)

File: docs/UserGuide.md (hunk 3 above)
Comment (KamJiaYue) [reply docs/UserGuide.md:100]: Agreed.

File: src/main/java/seedu/address/model/person/Phone.java (hunk 2 above)
Comment (KamJiaYue) [reply src/main/java/seedu/address/model/person/Phone.java]: You're right! Nice catch and thanks for informing.

**Review by laurenlim2112 — APPROVED** [review]
LGTM with the updates!

**Pull request merged**